### DISCUSSION TITLE
Rpi 4 add wifi dongle TL-WN725N

### DISCUSSION THREAD

**Discussion by Andromeda1966** [Ysurac/openmptcprouter#1873] [Q&A]
Hi,
I want to add a wifi usb dongle the TL-WN725N.
I have drivers from here, but never install drivers on linux neither on openmptcprouter.
(Driver here: http://downloads.fars-robotics.net/wifi-drivers/8188fu-drivers/8188fu-5.4.83-1379.tar.gz)

I got this on kernel log:

[  245.819823] usb 1-1.4: new high-speed USB device number 4 using xhci_hcd
[  245.924338] usb 1-1.4: New USB device found, idVendor=0bda, idProduct=8179, bcdDevice= 0.00
[  245.932691] usb 1-1.4: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[  245.940001] usb 1-1.4: Product: 802.11n NIC
[  245.944180] usb 1-1.4: Manufacturer: Realtek
[  245.948446] usb 1-1.4: SerialNumber: 00E04C0001
[  245.953164] usb 1-1.4: no of_node; not parsing pinctrl DT

May you help me to add it to awesome openmptcprouter, because I need to connect to 2 wifi router ?

Thanks

**Comment by Andromeda1966** [comment]
Any suggestion ?

Thanks in advance

**Reply by Ysurac** [reply]
This chipset doesn't seems to be supported by rtl8xxxu driver and I don't find driver support for OpenWrt so this seems difficult to support this.

**Reply by Andromeda1966** [reply]
Ok thanks.
Did you known wifi dongle supported ?
I mean a plug and play one, without do anything ?

Thanks in advance

**Reply by Andromeda1966** [reply]
Any suggestion ?
Thanks

**Reply by Ysurac** [reply]
I don't use any wifi dongle so I can't really help you, but you can look at OpenWrt forum.
If you want to see a list of drivers installed by default on OpenMPTCprouter you can look at https://github.com/Ysurac/openmptcprouter-feeds/blob/develop/openmptcprouter-full/Makefile (it's the "kmod-*" packages)
